### DISCUSSION TITLE
Battery level shows as lightbulb

### DISCUSSION THREAD

**Discussion by dimkagithub** [merdok/homebridge-miot#437] [Q&A] [answered]
Hi!
Dreame D9
Battery level shows as lightbulb. How can I change it to Battery Service?

**Comment by merdok** [comment]
That is quite impossible. Can you post a screenshot?

**Reply by dimkagithub** [reply]
Hi!
I created Monitor Property
```
      {
          "property": "battery:battery-level",
          "name": "Battery"
      }
```

And It shows as light sensor:

https://paste.pics/LVCWJ

**Reply by merdok** [reply]
Yes, this is expected as battery level does not have its own accessory but is a characteristic. In order to show the value anyway a light sensor is used to display the numeric battery level.

**Reply by dimkagithub** [reply]
Ok.

**Comment by dimkagithub** [comment]
Now I broke something.
Before when I tapped any room (kitchen, bedroom, etc) the Dreame starts to clean. But now it doesn't start.

https://paste.pics/LVDD2

Now it starts cleaning only if I tapped main switch (the first button on screenshot)

What can be wrong?

**Comment by merdok** [comment, marked as answer]
Sorry, I do not know what you changed so I cannot really help you...

**Reply by dimkagithub** [reply]
Hi!
It is ok. It works great! I just forgot to add this:

```
{
    "piid": 1,
    "value": 18
}
```
